personality/ui/rpms/config.pan

```diff
@@ -1,4 +1,4 @@
 unique template personality/ui/rpms/config;
 
 # EMI UI
-"/software/packages/{emi-ui}" ?= nlist();
```

Comment:
The name of the package depends on the OS release.
* For SL6: emi-ui
* For CentOS 7: ui

You can use the following fix:
```
"/software/packages" = {
    if (OS_VERSION_PARAMS['major'] == 'sl6') {
        pkg_repl('emi-ui');
    } else {
        pkg_repl('ui');
    };

    SELF;
};
```

Reply:
My bad, I modify the code as you suggested. Thanks.